packages/swingset-runner/src/main.js

```diff
@@ -368,6 +377,56 @@ export async function main() {
       case '--sbench':
         swingsetBenchmarkDriverPath = argv.shift();
         break;
+      case '--cpu-profile-output':
+        cpuProfileFilePath = path.resolve(String(argv.shift()));
+        await new Promise(resolve => {
+          const parentDirectory = path.dirname(String(cpuProfileFilePath));
+          fs.access(parentDirectory, fs.constants.F_OK, err => {
+            if (err) {
+              console.warn(
```

Comment:
This should throw rather than warn. Also, wouldn't the whole thing be easier with fs/promises?
```js
        try {
          const parentDirectory = path.dirname(cpuProfileFilePath);
          const dir = await fsp.opendir(parentDirectory);
          dir.closeSync();
          if (!inspectorSession) {
            await initializeInspectorSession();
          }
        } catch (cause) {
          throw Error(`Can't record CPU profile at ${cpuProfileFilePath}`, { cause });
        }
```